OrbitCore/SamplingProfiler.cpp

```diff
@@ -75,6 +75,7 @@ std::unique_ptr<SortedCallstackReport> SamplingProfiler::GetSortedCallstackRepor
 }
 
 const int32_t SamplingProfiler::kAllThreadsFakeTid = -1;
+const int32_t SamplingProfiler::kAllTracepointsFakeTid = -3;
```

Comment:
This doesn't really belong in `SamplingProfiler`, I would assume it's better to have it next to `kNotTargetProcessThreadId` in `TracepointEventBuffer`.

Reply:
Agree, if there isn't a *really* good reason the constants for the  fake id's should be in the same place.

Reply:
Absolutely! Thank you!